source/_posts/Write-the-first-Vue-js-Component-Unit-Test-in-Jest.md

```diff
@@ -257,15 +257,15 @@ If you're curious about **how shallow render is implemented**, check out the [so
 In the same vein, you can implement the `MessageList.test.js` test as follows:
 
 ```javascript
-import { shallowMount } from 'vue-test-utils'
+import { mount } from '@vue/test-utils'
```

Comment:
Here we need to use just `mount`, so that all the tests have passed..

See also https://github.com/alexjoverm/vue-testing-series/pull/22